website/static/js/share/sideBar.js

```diff
@@ -73,22 +74,25 @@ SideBar.controller = function(vm) {
                         utils.updateFilter(self.vm, 'source:' + result.short_name);
                     }
                 }
-            }, result.long_name
-        ));
+            }, [
+                m('img', {src: result.favicon, style: {height:'16px', width:'16px'}}), ' ', result.long_name
+            ])
+        );
 
     };
 
-    self.sortProviders = function() {
+    self.vm.sortProviders = function() {
```

Comment:
Since this is now a method of `vm`, should this method be moved to the viewmodel's definition?